app/scripts/components/common/page-header/nav/nav-item-cta.tsx

```diff
@@ -5,23 +5,29 @@ import { ActionNavItem } from '../types';
 
 interface NavItemCTAProps {
   item: ActionNavItem;
+  customClasses?: string;
 }
 
 export const NavItemCTA: React.FC<NavItemCTAProps> = ({
-  item
+  item,
+  customClasses
 }): JSX.Element => {
   const { isRevealed, show, hide } = useFeedbackModal();
   return (
     <React.Fragment key={item.id}>
       {item.actionId === 'open-google-form' && (
         <>
           <button
-            className='usa-nav__link'
+            className={
+              customClasses && customClasses != ''
+                ? customClasses
+                : 'usa-nav__link'
+            }
             type='button'
             tabIndex={0}
             id={item.id}
             onClick={show}
-            style={{background: 'none', border: 'none', cursor: 'pointer'}}
+            style={{ background: 'none', border: 'none', cursor: 'pointer' }}
```

Comment:
what was the reason for using inline styles?

Reply:
This existed previously with pageheader PR - (commit [here](https://github.com/NASA-IMPACT/veda-ui/pull/1247/commits/7652f0b129c7e6e7b7521d66ac491d35e8624342) to address this comment [here](https://github.com/NASA-IMPACT/veda-ui/pull/1247#discussion_r1869149971)). So removes button default styling but feel free to move it elsewhere 👍🏼 .